src/Dashboard/Data/FunctionInstanceSnapshot.cs

```diff
@@ -85,7 +85,11 @@ private string BuildFunctionDisplayTitle()
         {
             var name = new StringBuilder(FunctionShortName);
             IEnumerable<string> argumentValues = Arguments.Values.Select(v => v.Value);
-            string parametersDisplayText = String.Join(", ", argumentValues);
+
+
+            // Remove newlines to avoid 403/forbidden storage exceptions when saving display title to blob metadata
+            // for function indexes. Newlines may be present in JSON-formatted arguments.
+            string parametersDisplayText = String.Join(", ", argumentValues).Replace("\r\n", String.Empty);
```

Comment:
Do you think it would be better to replace newlines with a space rather than Empty? At least that preserves some separation.